### DISCUSSION TITLE
Map selecton filter

### DISCUSSION THREAD

**Discussion by Mr-Bajs** [ihhub/fheroes2#7586] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Not a big issue but.

When you start a new game and is choosing a map, it would be great if there where some sort of filtering and or sorting of the maps by Win/Lose condition. Though you can filter by map size but Win/Lose Condition would be nice, preferable by check boxes which is easy for both Desktop and Touch UI.

### Describe the possible solution

.

**Comment by ihhub** [comment]
This idea requires a lot of UX design ideas as it is not ready for implementation. Therefore, I am moving it to the discussion section.
